Modules/_datetimemodule.c

```diff
@@ -5199,7 +5199,8 @@ datetime_utcnow(PyObject *cls, PyObject *dummy)
     if (PyErr_WarnEx(PyExc_DeprecationWarning,
         "datetime.datetime.utcnow() is deprecated and scheduled for removal in a "
         "future version. Use timezone-aware objects to represent datetimes "
-        "in UTC: datetime.datetime.now(datetime.UTC).", 1))
+        "in UTC: datetime.datetime.now(datetime.UTC). To maintain compatibility"
+        "datetime.datetime.now(datetime.UTC).replace(tzinfo=None) may be used.", 1))
```

Comment:
It looks like this will have two words joinedtogether.

But do the warning messages really need to specify replacement options at all?